server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/DSNBounce.java

```diff
@@ -454,6 +453,6 @@ private MimeBodyPart createAttachedOriginal(Mail originalMail, TypeCode attachme
 
     @Override
     public MimeMessageModifier getMimeMessageModifier(Mail newMail, Mail originalMail) throws MessagingException {
-        return new MimeMessageModifier(originalMail.getMessage());
+        return new MimeMessageModifier(newMail.getMessage());
```

Comment:
Why ?

Reply:
This method is used to add the prefix to the subject of the message, so the modification is to the new email and not to the original email.

Reply:
Thanks for the explanation!